src/Game.java

```diff
@@ -0,0 +1,98 @@
+public class Game {
+
+    public Player[] players;
+
+    public Board board ;
+    private final int playerNumber;
+
+    public Game(int x) {
+        playerNumber = x;
+        players = new Player[x];
+        board = new Board();
+        if ( x >= 1 )
+            players[0] = new Player(Colour.GREEN, 0, 56);
+        if ( x >= 2 )
+            players[1] = new Player(Colour.BLUE, 28, 27);
+        if ( x >= 3)
+            players[2] = new Player(Colour.RED, 14, 13);
+        if ( x == 4)
+            players[3] = new Player(Colour.YELLOW, 42, 41);
+    }
+
+    public void round() {
+        for (int i = 0 ; i < playerNumber ; i++ ) {
+            players[i].playerMove();
+            checkBoard(players[i]);
+        }
+    }
+
+    private void checkSpecialFields(Player player) {
+        for (int i = 0 ; i < 4 ; i++ ) {
+            if ( player.pawns[i].getStatus() == PawnStatuses.IN_GAME) {
+                SpecialFieldTypes specialFieldType = board.getSpecialField((player.pawns[i].getPosition() + player.getFirstField())%61);
+                triggerSpecialFields(player.pawns[i], specialFieldType);
+                for ( int j = 0 ; j < playerNumber ; j++ ) {
+                    checkCollisionPlayer(player, players[j]);
+                }
+            }
+        }
+    }
+
+    private int teleportPawn() {
+        while (true) {
+            int x = (int) (Math.random() * 56);
+            if ( board.getSpecialField(x) == null ) {
+                return x;
+            }
+        }
+    }
```

Comment:
Teleport pawn miał teleportować na z góry wskazane miejcse, zapisane w klasie pola specjalnego teleport, nie na losowe

Reply:
![image](https://github.com/Lecrut/Chinczyk/assets/118774312/75557d21-f974-45c3-a69e-72cf9ec5bf83)

Reply:
losowe to losowe

Reply:
ok